Makefile

```diff
@@ -102,7 +102,9 @@ generate:
 .PHONY: generate-swagger
 generate-swagger:
 	@hash swagger > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
+		cd .. ; \
 		GO111MODULE="on" $(GO) get -u github.com/go-swagger/go-swagger/cmd/swagger@v0.20.1; \
+		cd gitea ; \
```

Comment:
```suggestion
		cd - ; \
```

Reply:
@lafriks I think that might not work with some limited versions of shell. Currently Linux's `/bin/sh` seems to handle it, but I do remember in the old days AIX or Solaris woudn't like it. But may be I'm too old.

Reply:
It's still better to have such name as otherwise what if I have cloned project in other directory than `gitea`?